src/main/resources/anniv5Resources/localization/eng/pixiepack/Relicstrings.json

```
@@ -0,0 +1,9 @@
+{
+  "${ModID}:DimensionCore": {
+    "NAME": "Dimension Core",
+    "FLAVOR": "A crystal sphere containing infinite possibilities. It's cracked.",
+    "DESCRIPTIONS": [
+      "At the start of combat for every 10 cards in your deck shuffle 3 foreign cards into your drawpile. Draw an extra card each turn."
```

Since you switched to using "different color", you should switch here too. Writing it as #ydifferent #ycolor will also give you highlighting, and putting a #b before the numbers will make them blue. Here's what I'd suggest the wording be.
```suggestion
      "At the start of combat, for every #b10 cards in your deck, shuffle #b3 #ydifferent #ycolor cards into your draw pile. Draw an extra card each turn."
```